app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+    @posts = @user.posts
+  end
+
+  def post
+    @user = User.find(params[:id])
+    @posts = @user.posts
+    @comments = @user.comments
+    @comments_count = @user.comments.count
+    @likes_count = @user.likes.count
+  end
+
+  def detail
+    @user = User.find(params[:id])
+    @posts = @user.posts
+    @comments = @user.comments
+    @comments_count = @user.comments.count
+    @likes_count = @user.likes.count
+    @idp = params[:idp]
+  end
```

Comment:
- [ ] [Optiona and Important] There is no need for these methods in your user controller, what you need now is just index and show, Kindly remove

Reply:
removed.

Reply:
![image](https://user-images.githubusercontent.com/13853450/192113170-32ab6b21-e956-49a5-9d84-ec6577531bc8.png)

Removed from  user. added to post